test/integration/encrypted_env_variables_test.js

```diff
@@ -37,7 +37,7 @@ suite('encrypted private env variables', () => {
 
   function getEncryptedEnvPayload(payloadData) {
 
-    return Promise.all(_.map(payloadData, (data) => {
+    return Promise.all(payloadData.map((data) => {
```

Comment:
Could you have a look at the test files where you removed lodash and see if the import of lodash could be removed?